configuration/rules-blockly-run-and-process.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: documentation
+title: Rules Blockly - Run & Process
+---
+
+# Run & Process (Rules and Scripts)
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+## Introduction
+
+This section contains several possibilities
+* Calling Rules or other scripts either located as files on the server or provided via the UI
```

Comment:
Please use `-` for lists in all files.

Currently we have mixed usage of `-` and `*`.